src/pricing.ts

```diff
@@ -476,6 +477,46 @@ const recomputeDetailTotalsFromCompositePrice = (
   }, details || initialPricingDetails);
 };
 
+const BLACK_LISTED_KEYS: Set<keyof Price> = new Set([
+  '_org',
+  '_schema',
+  '_created_at',
+  '_updated_at',
+  '_owners',
+  '_acl',
+  '_acl_sync',
+  '_viewers',
+  '_relations',
+  '$relation',
+  'file',
+  '_files',
+  'workflows',
+  '_slug',
+]);
+
+/**
+ * Maps the whitelisted keys from the _price that we want available, avoiding having unnecessary data in the metadata.
+ *
+ *? @todo We should define _price as not being an optional key in PriceItemDto since there is no case where is optional
+ *? Additionally, we should also define the _product as a separate interface to avoid `PriceItemDto['_product']` type definitions.
```

Comment:
```suggestion
 * Converts a Price entity into a PriceDTO without all fields present on the entity fields exclusion list.
```

I would add this TODO within the method, so it doesn pop up on the IDE intelisense.
```
 *? @todo We should define _price as not being an optional key in PriceItemDto since there is no case where is optional
 *? Additionally, we should also define the _product as a separate interface to avoid `PriceItemDto['_product']` type definitions.
```